docs/whatsnew/3.2.rst

```diff
@@ -168,12 +192,22 @@ following examples illustrate the new syntax for getting help::
   >>> Table.write.help('csv')
   >>> Table.read.help()  # Generic read help
 
+Deprecated/Renamed/Removed functionality
+========================================
+
+The bundled version of the `six <https://pypi.org/project/six/>`_ package in
```

Comment:
maybe we should also mention the removal of `pytest_plugins`, it'll bite downsteam packages.